src/components/app/list-new/AppListFilters.tsx

```diff
@@ -48,6 +48,20 @@ const AppListFilters = ({
         isExternalFlux,
     })
 
+    const getIsClusterOptionDisabled = (option: SelectPickerOptionType): boolean => {
+        const clusterList = appListFiltersResponse?.isFullMode
+            ? appListFiltersResponse?.appListFilters.result.clusters
+            : appListFiltersResponse?.clusterList.result
+        if (!clusterList) return false
```

Comment:
```suggestion
        if (!clusterList || (!isExternalArgo && !isExternalType)) return false
```

this will remove the next if check and extra return false

add comments as well or better make variables for condition